lib/workers/repository/process/lookup/index.ts

```diff
@@ -131,8 +134,9 @@ function getBucket(config: LookupUpdateConfig, update: LookupUpdate): string {
 }
 
 export async function lookupUpdates(
-  config: LookupUpdateConfig
+  inconfig: LookupUpdateConfig
 ): Promise<UpdateResult> {
+  let config: LookupUpdateConfig = { ...inconfig };
```

Comment:
No longer revert this?

Reply:
ok, we get eslint error because of param reassign 🤔  so leaving it as is here.

Reply:
![image](https://user-images.githubusercontent.com/3850197/99252792-1b8b5200-2842-11eb-9be0-88c20dcd08c5.png)

Reply:
Yes, that was my mistake because I forgot the reassignment, so I deleted the suggestion.